murdock/murdock.py

```diff
@@ -9,6 +9,7 @@
 import websockets
 from fastapi import WebSocket
 from prometheus_fastapi_instrumentator import Instrumentator
+import prometheus_client
```

Comment:
Nit: this import should be placed before "import websocket". In general, I try to follow the [PEP8 rule about imports](https://peps.python.org/pep-0008/#imports):

```
Imports should be grouped in the following order:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.
```

And also use `import something` before `from something import anothersomething`. This is usually raised by pylint linter (which is not configured in this project I admit)